src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2716,6 +2716,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
         End Function
 
+        Public Overrides Function GetRequiredLanguageVersion(diagnostic As Diagnostic) As String
+            ' The driving scenario for this API (upgrading project based on language versions reported in diagnostics)
```

Comment:
`AbstractDiagnosticPropertiesService` is language-agnostic and doesn't know that VB isn't implemented yet.
It could be fixed (by try/catching), but that seems less desirable.